src/folders.js

```diff
@@ -0,0 +1,34 @@
+const client = require('./lib/client')
+const log = require('./lib/logger')
+
+const operations = {
+  get: async (folderId) => {
+    const folder = await client.folders.get(folderId)
+    log(folder)
+    return folder
+  },
+  create: async (name, { parentID }) => {
+    const folder = await client.folders.create(parentID, name)
+    log(folder)
+    return folder
+  },
+  update: async (folderId, { name, description }) => {
+    const folder = await client.folders.update(folderId, { name, description })
+    log(folder)
+    return folder
+  },
+  delete: async (folderId) => {
+    await client.folders.delete(folderId)
+    log('Folder deleted')
+    return 'Folder deleted'
+  }
```

Comment:
Consider adding explicit error handling here. If `client.folders.delete(folderId)` throws an error (e.g., folder not found, permission issue), it might be better to catch it, log a more specific error message, and potentially re-throw or return an error indicator.